framework/src/main/java/org/checkerframework/common/wholeprograminference/scenelib/ASceneWrapper.java

```diff
@@ -48,7 +45,7 @@
 public class ASceneWrapper {
 
     /** The AScene being wrapped. */
-    private final AScene theScene;
+    public final AScene theScene;
```

Comment:
There is a getter for this field, so there is no reason to make it public. Can you return it to private and use the getter instead?